network/vpc/how-to/attach-resources-to-pn.mdx

```diff
@@ -137,5 +137,5 @@ You can also detach a resource to a Private Network from the resource's own sect
 
 <Navigation title="See also">
   <PreviousButton to="/network/vpc/how-to/activate-dhcp/">How to activate DHCP on a legacy Private Network</PreviousButton>
-  <NextButton to="/network/vpc/how-to/delete-private-network/">How to delete a Private Network</NextButton>
-</Navigation>
\ No newline at end of file
+  <NextButton to="/network/vpc/how-to/manage-routing/">How to manage routing on a VPC</NextButton>
```

Comment:
Considering the PR from Aymar, do we still keep the See also block here?

Reply:
I think we do until the new system is put in place